sdk/storage/azure-storage-blob/tests/recordings/test_common_blob_async.test_create_blob_with_requests_async.yaml

```diff
@@ -0,0 +1,11811 @@
+interactions:
```

Comment:
Do we really need this large recording? Our repo is bloating waaay to much right now.

Reply:
Hmm, if we want to run this test :)

Reply:
just noticed how big they are...removed and skipping for recording